pegasus/sites.v3/code.org/public/css/starwars.css

```diff
@@ -82,18 +82,21 @@
   }
 
   .starwars-tutorial {
-    width: 50%;
-    float: left;
+    width: 100%;
     height: 220px;
     padding: 5px;
-    padding-top: 10px;
     position: static;
+    margin-bottom: 10px;
   }
 
   .starwars-container .tutorial-box {
     background-image: url(/images/fit-520/starwars_carousel_background.jpg);
   }
 
+  .dance-party-container .tutorial-box {
```

Comment:
does dance party have its own CSS file? if so, these styles should go in there instead of `starwars.css` (or we rename this stylesheet, maybe?). this can be part of clean-up later and not ship-blocking